viadot/tasks/sap_rfc.py

```diff
@@ -97,6 +110,9 @@ def run(
             multiple options are automatically tried. Defaults to None.
             replacement (str, optional): In case of sep is on a columns, set up a new character to replace
                 inside the string to avoid flow breakdowns. Defaults to "-".
+            sap_credentials (dict, optional): The credentials to use to authenticate with SAP. By default, they're taken from the local viadot config.
```

Comment:
In this case by default it is taken from KeyVault (lines 136-141). Default is None and it is taken from KeyVault here in task or local config on the 'source' level (SAPRFC class). Please take a look and adjust docstring.